priv/repo/migrations/20240527155639_create_error_tracker_tables.exs

```diff
@@ -0,0 +1,6 @@
+defmodule ErrorTracker.Repo.Migrations.CreateErrorTrackerTables do
```

Comment:
Can we move this somehow to `/test/support`?

Just asking because in `/priv` it will be included in releases by default 😅

Reply:
I put it here because I was planning on using it from `dev.exs` as well. In the `mix.exs` file we declare just `priv/static` to be included in the release so I think that this file won't be included.

Reply:
I've just updated the `dev.exs` script so it uses the same migrations that are present in `priv/repo/`. This is consistent with how regular projects work and allows us to have a single source of truth for local migrations either in development or test.

I've also confirmed that `priv/repo/` is excluded from the build and won't be included when publishing new releases.